### DISCUSSION TITLE
Cannot connect Develco AQSZB-110 using ITSTUFF CC2531 ZigBee

### DISCUSSION THREAD

**Discussion by costasfait** [Koenkk/zigbee2mqtt#11307] [Devices]
Hi,
it seems that the Develco AQSZB-110 air-quality sensor is not supported after all, or is there any special configuration I should do.
I am using ITSTUFF CC2531 ZigBee dongle in case this might afect the verdict.
I trying using the 'edge' version and I could not properly connect with that as well.

would appreciate any help.

**Comment by workinregress** [comment]
I have the same issue. I have two of this type of device, but any of them is recognised. :(

**Comment by costasfait** [comment]
The plug in has changed repository. The new repository has an updated version that fixes the issue

**Reply by workinregress** [reply]
what do you mean? sorry

**Reply by workinregress** [reply]
where can I find the new repository? I'm running 1.29.0 version

**Comment by costasfait** [comment]
https://github.com/zigbee2mqtt/hassio-zigbee2mqtt
i use this repository and the develco device works well with v1.29.1-1

**Reply by workinregress** [reply]
I think I have no luck, I've tried several times to pair both devices, one is detected but not recognised, the other one is not even detected. If I could have get at least one paired I could have copy the settings from database.db and changing the address, but nothing T_T
